api/envoy/extensions/transport_sockets/tls/v3/common.proto

```diff
@@ -87,6 +89,24 @@ message TlsParameters {
   //   ECDHE-RSA-AES256-SHA
   //   AES256-GCM-SHA384
   //   AES256-SHA
+  //
+  // In non-FIPS builds, the default client cipher list is:
+  //
+  // .. code-block:: none
+  //
+  //   [ECDHE-ECDSA-AES128-GCM-SHA256|ECDHE-ECDSA-CHACHA20-POLY1305]
+  //   [ECDHE-RSA-AES128-GCM-SHA256|ECDHE-RSA-CHACHA20-POLY1305]
+  //   ECDHE-ECDSA-AES256-GCM-SHA384
+  //   ECDHE-RSA-AES256-GCM-SHA384
+  //
+  // In builds using :ref:`BoringSSL FIPS <arch_overview_ssl_fips>`, the default client cipher list is:
+  //
+  // .. code-block:: none
+  //
+  //   ECDHE-ECDSA-AES128-GCM-SHA256
+  //   ECDHE-RSA-AES128-GCM-SHA256
+  //   ECDHE-ECDSA-AES256-GCM-SHA384
+  //   ECDHE-RSA-AES256-GCM-SHA384
```

Comment:
It would be nice if we could generate these from some canonical location that is shared with code. If we aren't going to change frequently this is fine though.